src/person/models.py

```diff
@@ -7,6 +7,8 @@
 from prompt.models import Prompt
 from purpose.models import Purpose
 
+import datetime
```

Comment:
are you using `datetime` anywhere? I think you can delete

Reply:
this would've also been resolved with a pre-commit hook